### PR TITLE
[Doc] Tree-LSTM in DGL, edit pass for readability

### PR DESCRIPTION
**Pull request by john-andrilla**
Edit for grammar and style. 
How about deleting the last line? "Besides, you..." It needs a transition or some context.

### CODE REVIEW

File: tutorials/models/2_small_graph/3_tree-lstm.py [hunk 1]
```diff
@@ -370,7 +372,6 @@ def batcher_dev(batch):
             epoch, step, loss.item(), acc))
 
 ##############################################################################
-# To train the model on full dataset with different settings(CPU/GPU,
-# etc.), please refer to our repo's
-# `example <https://github.com/dmlc/dgl/tree/master/examples/pytorch/tree_lstm>`__.
-# Besides, we also provide an implementation of the Child-Sum Tree LSTM.
+# To train the model on a full dataset with different settings (such as CPU or GPU),
+# please refer to the `PyTorch example <https://github.com/dmlc/dgl/tree/master/examples/pytorch/tree_lstm>`__.
+# Besides, you also provide an implementation of the Child-Sum Tree-LSTM.
```
Comment (yzh119) [inline tutorials/models/2_small_graph/3_tree-lstm.py]: it's DGL-team that provide an implementation of the Child-Sum Tree-LSTM instead of user.

Comment (jermainewang) [reply tutorials/models/2_small_graph/3_tree-lstm.py]: @john-andrilla Would you mind changing the last sentence to "Besides, there is also an implementation of the Child-Sum Tree-LSTM." ? Thanks.

File: tutorials/models/2_small_graph/3_tree-lstm.py [hunk 2]
```diff
@@ -370,7 +372,6 @@ def batcher_dev(batch):
             epoch, step, loss.item(), acc))
 
 ##############################################################################
-# To train the model on full dataset with different settings(CPU/GPU,
-# etc.), please refer to our repo's
-# `example <https://github.com/dmlc/dgl/tree/master/examples/pytorch/tree_lstm>`__.
-# Besides, we also provide an implementation of the Child-Sum Tree LSTM.
+# To train the model on a full dataset with different settings (such as CPU or GPU),
+# please refer to the `PyTorch example <https://github.com/dmlc/dgl/tree/master/examples/pytorch/tree_lstm>`__.
```
Comment (john-andrilla) [inline tutorials/models/2_small_graph/3_tree-lstm.py]:
```suggestion
# refer to the `PyTorch example <https://github.com/dmlc/dgl/tree/master/examples/pytorch/tree_lstm>`__.
```

File: tutorials/models/2_small_graph/3_tree-lstm.py (hunk 1 above)
Comment (john-andrilla) [inline tutorials/models/2_small_graph/3_tree-lstm.py]:
```suggestion
# There is also an implementation of the Child-Sum Tree-LSTM.
```

Comment (john-andrilla) [reply tutorials/models/2_small_graph/3_tree-lstm.py]: Understood. addA

**Pull request merged**